src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.command.Command;
+import duke.task.TaskList;
+
+/**
+ * Duke is an automated chat-bot which can help you manage your tasks.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructs a Duke.
+     *
+     * @param filePath string representing the file path of a save file
```

Comment:
I noticed this a few times throughout the code, perhaps there should be a punctuation after the `filePath` param description, and the first letter of the description should be capitalised? [(Reference: Punctuation behind each parameter description)](https://se-education.org/guides/conventions/java/index.html#comments)

```suggestion
     * @param filePath String representing the file path of a save file.
```

Reply:
Thanks for helping me spot my mistakes!